src/domains/qi-common/env/weu-dev/terraform.tfvars

```diff
@@ -39,3 +42,25 @@ qi_storage_params = {
   public_network_access_enabled = true,
   access_tier                   = "Hot"
 }
+
+# SMO configuration 
+cidr_subnet_pg_flex_smo_dbms = ["10.1.193.0/24"]
+
+pgres_flex_smo_params = {
+  private_endpoint_enabled = false
+  sku_name                 = "GP_Standard_D4ds_v4"
+  db_version               = "13"
+  # Possible values are 32768, 65536, 131072, 262144, 524288, 1048576,
+  # 2097152, 4194304, 8388608, 16777216, and 33554432.
+  storage_mb                      = 32768
+  zone                            = 1
+  backup_retention_days           = 7
+  geo_redundant_backup_enabled    = false
+  create_mode                     = "Default"
+  high_availability_enabled       = false
+  standby_availability_zone       = 2
+  pgbouncer_enabled               = true
+  alerts_enabled                  = false
+  max_connections                 = 1000
+  enable_private_dns_registration = false
```

Comment:
@pasqualespica dev dns has been created (also for the other databases), so you can enable this property on dev, to keep all the envs aligned